projects/plugins/boost/app/rest-api/endpoints/List_Site_Urls.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Create a new request for site urls.
+ *
+ * Handler for GET '/list-site-urls'.
+ */
+
+namespace Automattic\Jetpack_Boost\REST_API\Endpoints;
+
+use Automattic\Jetpack_Boost\Lib\Site_Urls;
+use Automattic\Jetpack_Boost\REST_API\Contracts\Endpoint;
+use Automattic\Jetpack_Boost\REST_API\Permissions\Signed_With_Blog_Token;
+
+class List_Site_Urls implements Endpoint {
+
+	public function request_methods() {
+		return \WP_REST_Server::READABLE;
+	}
+
+	// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+	public function response( $request ) {
+		return rest_ensure_response( Site_Urls::get() );
+	}
```

Comment:
Pro-tip: We have this rule rigged so that any variable which starts with `$_` in the PHP is bypassed. So an alternate to this comment would be to rename `$request` to `$_request`.

Reply:
That's great - thanks! Is there a list of tips like this I could go through for future reference?

Reply:
No, I don't have an explicit tips guide. Maybe you could start one as you learn things? :)